package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-sdk-mock",
-  "version": "4.4.0",
+  "version": "4.4.1",
```

Comment:
version of package is updated by running the `npm version` script (_we don't update it manually_) 😉